engine/graphics/src/vulkan/graphics_vulkan_device.cpp

```diff
@@ -23,6 +23,7 @@ namespace dmGraphics
         t->m_Type                = TEXTURE_TYPE_2D;
         t->m_GraphicsFormat      = TEXTURE_FORMAT_RGBA;
         t->m_DeviceBuffer        = VK_IMAGE_USAGE_SAMPLED_BIT;
+        t->m_Format              = VK_FORMAT_UNDEFINED;
```

Comment:
Found this uninitialized variable in a comparison, since the value wasn't part of the `VkFormat` enum

Reply:
What could be the implication of having this value uninitialised?